### PR TITLE
Sum multiples

### PR DESCRIPTION
**Pull request by iivvaki**
Сума кратних

### CODE REVIEW

File: src/SumMultiples.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+public class SumMultiples {
+    public int sumOfMultiples(int n) {
+        int sum = 0;
+        if(n >= 1 && n <= 10000){
```
Comment (Lahutina) [inline src/SumMultiples.java:4]: Do you really need this if statement?

File: src/SumMultiples.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+            for(int i = 1; i < n+1; i++){
```
Comment (Lahutina) [inline src/SumMultiples.java:5]: for (int i = 1; i < n+1; i++) {
                if (i % 3 == 0 || i % 5==0 || i % 7==0) {
                    sum += i;
                }
            } 
Formating is very important. Try before commiting changes do ctrl+shift+alt+l

File: src/SumMultiples.java (hunk 1 above)
Comment (iivvaki) [reply src/SumMultiples.java:4]: maybe no

File: src/SumMultiples.java (hunk 2 above)
Comment (iivvaki) [reply src/SumMultiples.java:5]: do you mean i will do this? if (i % 3 == 0 || i % 5 == 0 || i % 7 == 0) sum += i;

**Pull request merged**